fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/Lao.java

```diff
@@ -243,27 +245,27 @@ public void setCreation(Long creation) {
     this.creation = creation;
   }
 
-  public void setOrganizer(String organizer) {
+  public void setOrganizer(PublicKey organizer) {
     this.organizer = organizer;
-    if (nodes.stream().noneMatch(node -> node.getPublicKey().equals(organizer))) {
+    if (nodes.stream().map(ConsensusNode::getPublicKey).noneMatch(organizer::equals)) {
```

Comment:
Yes sure

Reply:
@arnauds5 Actually, would it make sense to change the type of `consensuses` from `List<Consensus>` to `Map<MessageID, Consensus>` ?
Same for the `nodes` field in `Lao`

This should be done in a new PR but it would simplify those statements.